src/form/grid/Cell.jsx

```diff
@@ -390,6 +394,31 @@ const Cell = ({ node: el, form }) => {
             }}
           />
         );
+      case 'payment_method':
+        return (
+          <Elements.PaymentMethodField
+            {...fieldProps}
+            runtime
+            setCardElement={setCardElement}
+            setFieldError={async (message) => {
```

Comment:
async/await can be removed, and return setFormElementError from the function directly

Reply:
Yep, I refactored how I was doing this and left this in.  Fixed.